style.css

```diff
@@ -499,6 +572,10 @@ header {
     margin-bottom: 40px;
   }
 
+  /* .Desk_list a {
+    color: #6070ff;
+  } */
+
```

Comment:
## Recommendation Change

- [ ] This is not a major change to ask for, but I highly recommend you remove this commented code from your codebase. It's good to have a clean and professional code. Kindly remove this 👍🏽